<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>recepcionista_dialogo_gestionar_internado</class>
 <widget class="QDialog" name="recepcionista_dialogo_gestionar_internado">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>380</width>
    <height>329</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background: white;</string>
  </property>
  <widget class="QLabel" name="lbl_foto_paciente">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>20</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:none;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="../../resources.qrc">:/imagenes/Imagenes/imgPrueba.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>117</x>
     <y>200</y>
     <width>99</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Días Restantes:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_nombre_paciente">
   <property name="geometry">
    <rect>
     <x>96</x>
     <y>139</y>
     <width>201</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background:none;</string>
   </property>
   <property name="text">
    <string>Luis Rolando Banderas Barrera</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_medico">
   <property name="geometry">
    <rect>
     <x>176</x>
     <y>169</y>
     <width>92</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Luis Banderas</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_prioridad_medico_tarjeta_horario">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>230</y>
     <width>75</width>
     <height>16</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>75</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>75</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Montserrat</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Habitación:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>116</x>
     <y>169</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Medico:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="restantes_lineEdit">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>190</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
border: none;
border-bottom: 1px solid #818181;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="habitacion_lineEdit">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>220</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
border: none;
border-bottom: 1px solid #818181;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_agregar_internado_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>270</y>
     <width>120</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">*{
	background-color: white;
	color: #3E64FF;
	border: 1px solid #3E64FF;
	border-radius: 5px;
	padding: 5px;
}
</string>
   </property>
   <property name="text">
    <string>Guardar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_cancelar_internado">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>270</y>
     <width>120</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">*{
	background-color: #EF1F4B;
	color: white;
	border: 1px solid #EF1F4B;
	border-radius: 5px;
	padding: 5px;
}
</string>
   </property>
   <property name="text">
    <string>Dar de baja</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../../resources.qrc"/>
 </resources>
 <connections/>
</ui>
